src/SerializeAST.cc

```diff
@@ -823,66 +830,88 @@ void findNamedDataStructures(const snowcrash::Elements& elements, DataStructures
 
     }
 }
-#endif
 
-sos::Object drafter::WrapBlueprint(const Blueprint& blueprint, const ASTType astType)
+void registrateNamedMSONStructures(const snowcrash::Elements& elements, refract::Registry& registry) 
 {
-    sos::Object blueprintObject;
-
-#if _WITH_REFRACT_
     DataStructures found;
-    findNamedDataStructures(blueprint.content.elements(), found);
+    findNamedDataStructures(elements, found);
```

Comment:
No, but we have to rename it now since there was a renaming change in the
refract spec.

On Mon, Aug 17, 2015, 18:26 klokane <notifications@github.com> wrote:

> In src/SerializeAST.cc
> <https://github.com/apiaryio/drafter/pull/79#discussion_r37184316>:
>
> >      DataStructures found;
> > -    findNamedDataStructures(blueprint.content.elements(), found);
> > +    findNamedDataStructures(elements, found);
>
> BTW, this function is named findNamedDataStructures since May-29 will we
> rename it again in next three months?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apiaryio/drafter/pull/79/files#r37184316>.
>